flink-end-to-end-tests/test-scripts/elasticsearch-common.sh

```diff
@@ -17,8 +17,6 @@
 # limitations under the License.
 ################################################################################
 
-set -o pipefail
```

Comment:
are we assuming that every script should always include `test-scripts/common.sh` before anything else?

Reply:
If you take a look at two lines below: "Must run common.sh before elasticsearch-common.sh."

Reply:
yes, but I believe that to be a reasonable assumption. alternatively, could we include it in the runner scripts instead?

Reply:
So you mean to move the one and only `set` command to `test-runner-common.sh`?

Reply:
> So you mean to move the one and only set command to test-runner-common.sh

yes that was my thinking. Every test has to include it anyway.